Source/WebCore/page/Navigator.cpp

```diff
@@ -426,7 +426,7 @@ void Navigator::setAppBadge(std::optional<unsigned long long> badge, Ref<Deferre
         return;
     }
 
-    page->badgeClient().setAppBadge(page.get(), SecurityOriginData::fromFrame(frame.get()), badge);
+    page->badgeClient().setAppBadge(frame.get(), SecurityOriginData::fromFrame(frame.get()), badge);
```

Comment:
We don't need to early return if page is null any more.  page isn't used.

Reply:
`page` is used - that's where we get the badgeClient from.